Remove most of the PTQ receipe (#1219)
pytorch/torchtune: recipes/quantization.md

# Quantization and Sparsity

torchtune integrates with [torchao](https://github.com/pytorch/ao/) for architecture optimization techniques including quantization and sparsity. Currently only some quantization techniques are integrated, see the docstrings in the [quantization recipe](quantize.py) and the [QAT recipe](qat_distributed.py) for more details.
torchtune integrates with [torchao](https://github.com/pytorch/ao/) for QAT and QLoRA. Currently only some quantization techniques are integrated, see the docstrings in the [quantization recipe](quantize.py) and the [QAT recipe](qat_distributed.py) for more details.

#### Quantize
To quantize a model (default is int4 weight only quantization):
```

> tune run quantize --config quantization

Reviewer: note: this is still referenced in the QAT tutorial, so maybe we need to keep this in README but move it to under the QAT section?

> tune run quantize --config quantization

Reviewer: note: this is still referenced in the [QAT tutorial](https://pytorch.org/torchtune/main/tutorials/qat_finetune.html), so maybe we need to keep this in README but move it to under the QAT section?

Author: should we just link to QAT tutorial directly here?

Author: also I thought you are using `Int8DynActInt4WeightQuantizer` but not the quantization receipe?

> For post training quantization, User can use `torchao` directly: https://github.com/pytorch/ao/blob/main/torchao/quantization/README.md to quantize their model

Reviewer 2: ```suggestion
For post training quantization, we recommend using `torchao` directly: https://github.com/pytorch/ao/blob/main/torchao/quantization/README.md to quantize their model
```
